### ISSUE TITLE
Issues with config.yml

### ISSUE THREAD

**Issue by Evergrin** [pschou/snmp-prom#1]
Hello!
There are some issues with configuration file.
If I take the conf file as it is, just with my addresses and passwords, it doesn't pass the yaml-lint, because of the following error:
`yamllint config_basic.yml
config_basic.yml
  3:1       error    syntax error: expected <block end>, but found '-'
`
And if I insert the word 'names:' just before it (like following)
`---
interval: '30s'
names:
  - name: 'router'
    enabled: 'true'
`
the config file passes the yaml-lint test, but the container just exits with (0) exit code.
Is it the way the container supposed to behave?
Can you advice, how can I use it?

**Comment by pschou** [comment]
You are correct, wow, the example config file was missing a single line declaring the devices group and I failed to lint the config.

Please pull the changes I just pushed which includes a second example yml file, config_full.yml.

Please let me know how these work for you.

All the best!
